StochasticBarrierFunctions/src/system.jl

```diff
@@ -33,6 +33,30 @@ end
 dynamics(system::AdditiveGaussianLinearSystem) = (system.A, system.b)
 
 
+struct AdditiveGaussianPolySystem{T, N} <: AbstractAdditiveGaussianSystem{N}
+    # This struct represents a system with the dynamics x(k + 1) = f(x(k)) + v(k)
+    # where f is polynomial in x and the set of random variables (v(k))_{k ∈ ℕ} are
+    # independent and identically Gaussian distributed with zero mean and diagonal
+    # covariance. The stochasticity is represented by the diagonal std.dev.'s σ.
+
+    f::Any
```

Comment:
```suggestion
struct AdditiveGaussianPolySystem{T, N, P <: AbstractPolynomialLike} <: AbstractAdditiveGaussianSystem{N}
    # This struct represents a system with the dynamics x(k + 1) = f(x(k)) + v(k)
    # where f is polynomial in x and the set of random variables (v(k))_{k ∈ ℕ} are
    # independent and identically Gaussian distributed with zero mean and diagonal
    # covariance. The stochasticity is represented by the diagonal std.dev.'s σ.

    f::Vector{P}
```
Just a tiny improvement that avoid type instabilities. If including this, please do update the constructor accordingly.